users.go

```diff
@@ -816,7 +816,8 @@ func (s *UsersService) GetEmail(email int, options ...RequestOptionFunc) (*Email
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/projects.html#add-email
```

Comment:
```suggestion
// GitLab API docs: https://docs.gitlab.com/ce/api/users.html#add-email
```

Reply:
Just a question (I'm new in Golang): I've propose few pull requests in last weeks. I modified some source code, but I've never proposed documentation update. Is it auto-generated, or must I do something ?

Reply:
Docs are not generated, just added and maintained by hand...

Reply:
Ok
So, what can i do to help you to maintain the doc?

Reply:
Thanks, but not much to be honest. Of course make sure then when making a PR you add the correct links, but other then that not much...